packages/web/src/components/track/desktop/TrackTile.tsx

```diff
@@ -153,6 +156,9 @@ const TrackTile = ({
   const isLongFormContent =
     genre === Genre.PODCASTS || genre === Genre.AUDIOBOOKS
 
+  const isPurchase =
+    premiumConditions && isPremiumContentUSDCPurchaseGated(premiumConditions)
```

Comment:
don't think we need the `premiumConditions` check here?

Reply:
I would think so too! But it turns out on desktop, `premiumConditions` can be `undefined`, which isn't a valid input for the helper. I can, of course, update the helper to potentially accept undefined. Something about a function accepting undefined as a valid input has always bugged me :-)

Reply:
Updated my copy of the function to allow undefined. Will update the shared version when I merge the feature branch into this one.